src/redux/reducers/browser.ts

```diff
@@ -3,7 +3,6 @@ import {BrowserTypes} from '../types/actionTypes';
 import {WindowState} from '../types/BrowserState';
 import Action from '../types/redux';
 
-// Code smell on this function signature
 function windowSize(state: WindowState = {height: NaN, width: NaN}, action: Action) {
```

Comment:
This is how redux is designed to be used, this code smell needs to be disabled

Reply:
from what I see it is just saying to put the defaults at the end.
https://sonarcloud.io/project/issues?id=ius-csg_cslabs-webapp&issues=AXsiTV_Kw3kkXW2HHOS-&open=AXsiTV_Kw3kkXW2HHOS-

Reply:
Jason is right, if you try to flip it, it breaks redux. I marked this as a false positive.